andela/mrm_api
# CV3-68 Refactor fixtures folder

#### Description

Some fixtures files have more than 250 lines which is flagged by code climate as bad practice. Reorganise the fixture files by breaking big files into smaller ones.

#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
 - Check the `codeclimate` report to see if any file from the fixtures folder violates the `file-line-limit`
 - Follow the instructions on the `readme.md` file on running tests. Check to see if tests run successfully 

#### Checklist

#### JIRA
[CV3-68](https://andela-apprenticeship.atlassian.net/browse/CV3-68)
